data/crac/crac-util/src/main/java/com/farao_community/farao/data/crac_util/UcteNodeMatchingRule.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_util;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+
+public enum UcteNodeMatchingRule {
+
+    FIRST_7_CHARACTER_EQUAL(7),
+    ALL_8_CHARACTER_EQUAL(8),
+    BEGINNING_OF_ELEMENT_NAME(17);
```

Comment:
This one is a bit strange, is it supposed to be used potentially as a parameter of `createAliases`

Reply:
Moved